var/spack/repos/builtin/packages/amdblis/package.py

```diff
@@ -17,8 +17,9 @@ class Amdblis(BlisBase):
 
     _name = 'amdblis'
     homepage = "https://developer.amd.com/amd-aocl/blas-library/"
-    url = "https://github.com/amd/blis/archive/2.1.tar.gz"
+    url = "https://github.com/amd/blis/archive/2.2.tar.gz"
     git = "https://github.com/amd/blis.git"
 
+    maintainers = ['amd-toolchain-support']
+
     version('2.2', sha256='e1feb60ac919cf6d233c43c424f6a8a11eab2c62c2c6e3f2652c15ee9063c0c9')
-    version('2.1', sha256='3b1d611d46f0f13b3c0917e27012e0f789b23dbefdddcf877b20327552d72fb3')
```

Comment:
Is there any reason not to keep the old version? We generally keep old versions in packages in case someone needs to reproduce an older installation.

Reply:
In Spack, AOCC recipe support starts from 2.2 release.

To align with AOCC spack recipe,  AMD Optimizing CPU Libraries (AOCL) team decides to support all their libraries starting from 2.2 version onwards.